Python/Tests/DebuggerTests/AttachTests.cs

```diff
@@ -777,6 +777,8 @@ void main()
         [TestMethod, Priority(0)]
         [TestCategory("10s")]
         public virtual async Task AttachWithOutputRedirection() {
+            Version.AssertInstalled();
```

Comment:
You should add this to CheckVersion() at the top of this class instead.

Reply:
Oddly enough, this didn't actually fix the problem... still looking into it.